services/apps/premium/members_enrichment_worker/src/workflows/enrichMember.ts

```
@@ -1,121 +1,80 @@
-  if (organizations.length > 0) {
-    try {
-      await syncOrganizationsToOpensearch(organizations)
-    } catch (err) {
-      throw new Error(err)
+  if (changeInEnrichmentSourceData) {
+    console.log('Member enrichment data has been updated, use squasher again!')
+    const toBeSquashed = {}
+    for (const source of sources) {
+      // find if there's already saved enrichment data in source
+      const cache = await findMemberEnrichmentCache(source, input.id)
+      if (cache && cache.data) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)